bundles/org.openhab.binding.zwavejs/src/main/java/org/openhab/binding/zwavejs/internal/config/ZwaveJSChannelConfiguration.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.zwavejs.internal.config;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * The {@link ZwaveJSNodeConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Leo Siepel - Initial contribution
+ */
+@NonNullByDefault
+public class ZwaveJSChannelConfiguration {
+    public @Nullable String incomingUnit;
+    public String itemType = "String";
```

Comment:
Should you define "String" as default value for the parameter in XML definition ?